code/__DEFINES/flags.dm

```diff
@@ -27,6 +27,12 @@
 ///Whether or not this atom shows screentips when hovered over
 #define NO_SCREENTIPS			32768
 
+/// A surgical tool; when in hand in help intent (and with a surgery in progress) won't attack the user
+#define SURGICALTOOL 			65536
+
+/// An advanced surgical tool. If a surgical tool has this flag, it will be able to automatically repeat steps until they succeed.
+#define SURGICALTOOL_ADVANCED 	(1<<17)
```

Comment:
I had the impression that we shouldn't be adding any more new flags, and use traits for that purpose instead.

Reply:
If that's the case I can definitely move this over

Reply:
Hmm, double-checking, do we even have traits for tools? I'm not entirely sure what you mean.

Reply:
Talked with charlie about it, converted to traits in https://github.com/ParadiseSS13/Paradise/pull/18325/commits/b6505a81925ba572a766b5100fa8cc57cc5a08a0